src/main/java/com/questhelper/quests/sleepinggiants/SleepingGiants.java

```diff
@@ -0,0 +1,300 @@
+		speakToKovacForCommission.addStep(new Conditions(metalPoured, new Conditions(LogicType.NOR, bucket)), takeBucket);
+		speakToKovacForCommission.addStep(new Conditions(preform), forgeSword);
+
+		steps.put(25, speakToKovacForCommission);
+
+		return steps;
+	}
+
+	public void setupItemRequirements()
+	{
+		oakLogs = new ItemRequirement("Oak Logs", ItemID.OAK_LOGS);
+		oakLogs.setQuantity(3);
+
+		wool = new ItemRequirement("Wool", ItemID.WOOL);
+
+		nails = new ItemRequirement("Nails", ItemID.BRONZE_NAILS);
+		nails.addAlternates(ItemID.IRON_NAILS, ItemID.STEEL_NAILS, ItemID.BLACK_NAILS, ItemID.MITHRIL_NAILS, ItemID.ADAMANTITE_NAILS, ItemID.RUNE_NAILS);
+		nails.setQuantity(10);
+
+		hammer = new ItemRequirement("Hammer", ItemID.HAMMER);
+		hammer.canBeObtainedDuringQuest();
+
+		chisel = new ItemRequirement("Chisel", ItemID.CHISEL);
+
+		iceGloves = new ItemRequirement("Ice Gloves", ItemID.ICE_GLOVES);
+		iceGloves.setTooltip("Allows you to skip (filling and) using a bucket of water.");
+
+		bucket = new ItemRequirement("Bucket", ItemID.BUCKET);
+		bucket.canBeObtainedDuringQuest();
+		bucket.setHighlightInInventory(true);
+
+		bucketOfWater = new ItemRequirement("Bucket of Water", ItemID.BUCKET_OF_WATER);
+		bucketOfWater.canBeObtainedDuringQuest();
+
+		freeInventorySpace = new FreeInventorySlotRequirement(InventoryID.INVENTORY, 20);
+
+		alKharidTeleport = new ItemRequirement("Al Kharid Teleport", ItemCollections.getRingOfDuelings());
+		alKharidTeleport.addAlternates(ItemCollections.getAmuletOfGlories());
+		alKharidTeleport.setTooltip("Ring of Dueling or Amulet of Glory; Can be used to quickly reach the Giants' Foundry.");
+
+		twoOakLogs = new ItemRequirement("Oak Logs", ItemID.OAK_LOGS);
+		twoOakLogs.setQuantity(2);
+
+		fiveNails = new ItemRequirement("Nails", ItemID.BRONZE_NAILS);
+		fiveNails.addAlternates(ItemID.IRON_NAILS, ItemID.STEEL_NAILS, ItemID.BLACK_NAILS, ItemID.MITHRIL_NAILS, ItemID.ADAMANTITE_NAILS, ItemID.RUNE_NAILS);
+		fiveNails.setQuantity(5);
+
+		oneOakLog = new ItemRequirement("Oak Logs", ItemID.OAK_LOGS);
+
+		// TODO: This might not be the right Item ID.
+		preform = new ItemRequirement("Preform", 27010);
+		preform.equipped();
+	}
+
+	public void setupZones()
+	{
+		Zone desertPlateau = new Zone(new WorldPoint(3375, 3169, 0), new WorldPoint(3349, 3143, 0));
+		onDesertPlateau = new ZoneRequirement(desertPlateau);
+
+		Zone giantsFoundry = new Zone(new WorldPoint(3357, 11480, 0), new WorldPoint(3380, 11510, 0));
+		inGiantsFoundry = new ZoneRequirement(giantsFoundry);
+	}
+
+	public void setupConditions()
+	{
+		grindstoneFixed = new VarbitRequirement(13905, 2, Operation.EQUAL);
+		polishingStoneFixed = new VarbitRequirement(13906, 2, Operation.EQUAL);
+		hammerFixed = new VarbitRequirement(13904, 2, Operation.EQUAL);
+
+		commissionReceived = new VarbitRequirement(13903, 10, Operation.EQUAL);
+		crateSearched = new VarbitRequirement(13903, 25, Operation.LESS_EQUAL);
```

Comment:
```suggestion
		crateSearched = new VarbitRequirement(13903, 15, Operation.EQUAL);
```

Changing the Varbit value here should fix this step.